testsuite/Makefile

```diff
@@ -52,14 +52,18 @@ else # Windows
   endif
 endif
 
-ifeq "$(FLEXLINK_ENV)" ""
-  ocamltest := MKDLL="$(MKDLL)" SORT=$(SORT) MAKE=$(MAKE) $(ocamltest_program)
+ifeq "$(ocamltest_program)" ""
+  ocamltest = $(error ocamltest not found in $(ocamltest_directory))
```

Comment:
Out of curiosity: could you run this as a separate conditional first (without putting the rest in an extra `else .. endif` level), given that the build errors if it fires?

Reply:
I wanted to keep `make` working - i.e. the help target